ethcore/src/blockchain/blockchain.rs

```diff
@@ -536,7 +536,9 @@ impl BlockChain {
 		};
 
 		// load best block
-		let best_block_hash = match bc.db.key_value().get(db::COL_EXTRA, b"best").unwrap() {
+		let best_block_hash = match bc.db.key_value().get(db::COL_EXTRA, b"best")
+			.expect("Low level database error. Some issue with disk?")
```

Comment:
Maybe we could mention that we're trying to get the `best` block? 
"Low-level database error when fetching 'best' block." ?